pages/blueprint/index.js

```diff
@@ -278,6 +280,34 @@ class BlueprintPage extends React.Component {
     return `/cockpit/channel/${cockpit.transport.csrf_token}?${query}`;
   }
 
+  imageTypeToLabel(imageType) {
+    const imageTypeLabels = {
+      ami: "Amazon Machine Image Disk (.ami)",
+      "ext4-filesystem": "Ext4 File System Image (.img)",
+      "fedora-iot-commit": "Fedora IoT Commit (.tar)",
+      "live-iso": "Live Bootable ISO (.iso)",
+      "partitioned-disk": "Raw Partitioned Disk Image (.img)",
+      qcow2: "QEMU QCOW2 Image (.qcow2)",
+      openstack: "OpenStack (.qcow2)",
+      "rhel-edge-commit": "RHEL for Edge Commit (.tar)",
+      tar: "TAR Archive (.tar)",
+      vhd: "Azure Disk Image (.vhd)",
+      vmdk: "VMware Virtual Machine Disk (.vmdk)",
+      alibaba: "Alibaba Machine Image (.qcow2)",
+      google: "Google Compute Engine Image (.tar.gz)",
+      "hyper-v": "Hyper-V Virtual Machine Disk (.vhdx)",
+    };
+    return imageTypeLabels[imageType];
+  }
```

Comment:
We currently have a list of image types and their labels in [composer.js](https://github.com/osbuild/cockpit-composer/blob/master/core/composer.js#L132). Would it be possible to have this list of labels defined in one place that is referenced by every page/component that needs it?

Also, in composer.js, there is a conditional to handle the case where a new type has been added to osbuild-composer but hasn't been added to this list yet. If we do keep separate lists for defining labels, it might be good to add something like that here.